src/components/shared/generalText.js

```diff
@@ -1,35 +1,56 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { graphql } from 'gatsby';
-
-const GeneralText = (props) => {    
-    let processed = props.processed;
-    let textClass = props.textClass;    
-    return <div {...(textClass !== `` ? {className:textClass} : {})} dangerouslySetInnerHTML={{__html: processed}}></div>    
-}
-
-GeneralText.propTypes = {
-    processed: PropTypes.string,
-    textClass: PropTypes.string,
-}
-
-GeneralText.defaultProps = {
-    processed: ``,
-    textClass: ``,
-}
-
-export default GeneralText
-
-export const query = graphql`
-  fragment GeneralTextParagraphFragment on paragraph__general_text {
-    drupal_id
-    field_general_text {
-      processed
-    }
-    relationships {
-      field_section_column {
-        name
-      }
-    }
-  }
-`
\ No newline at end of file
+import React from "react"
+import PropTypes from "prop-types"
+import { graphql, Script } from "gatsby"
+import { Parser, ProcessNodeDefinitions } from "html-to-react"
+
+const GeneralText = (props) => {
+  const parser = new Parser()
+  const instructions = [
+    {
+      // Replace <script> tags with Gatsby <Script> components
+      shouldProcessNode: (node) => node.name === "script" && node.attribs?.src,
+      processNode: (node) => <Script src={node.attribs.src} />,
+    },
+    {
+      // Inline scripts will also be replaced but should only have one child, which is the script text
+      shouldProcessNode: (node) => node.name === "script" && node?.children.length === 1,
+      processNode: (node) => <Script>{node.children[0].data}</Script>,
+    },
+    {
+      // Process all other nodes with the default parser
+      shouldProcessNode: () => true,
```

Comment:
Does this mean everything gets converted to React elements? So far, all the pages I've checked seem to appear as expected, so that's not necessarily a bad thing. But I am curious...

Reply:
Yeah it does, as far as I can tell it shouldn't have any negative/unexpected effects.

Reply:
I have an idea for how we could work around it so that only the script tags are converted to React elements but ill only do that if it actually becomes an issue

Reply:
I'm fine with that approach, especially because there might be future situations where the conversion comes in handy (e.g. inline images)